themes/blocky.ps1

```diff
@@ -2,8 +2,9 @@
     $realCommandStatus = $?
     $realLASTEXITCODE = $LASTEXITCODE
     $Path = $pwd.ProviderPath
+    Write-Host
     Write-Host " $Path " -NoNewLine -ForegroundColor Black -BackgroundColor White
-    if($gitStatus){
+    if(Get-GitStatus){
```

Comment:
This will cause to many unnecessary calls to the Get-Gitstatus function, $gitstatus represents if git-posh exists or not.